### DISCUSSION TITLE
How to download Tailwind CSS Gradients?

### DISCUSSION THREAD

**Discussion by montasim** [STRINGLABITSOLUTIONS/Bhojon#198] [Q&A] [answered]
In short.

**Comment by fcasamento** [comment]
@montasim Can you describe to me what this download gradients would be in order to help.

**Comment by hongducdev** [comment, marked as answer]
Use npm install tailwindcss-gradients
See details at: https://www.npmjs.com/package/tailwindcss-gradients
Or: https://github.com/benface/tailwindcss-gradients

You can see how to make gradient background using Tailwind CSS at Tailwind CSS homepage: https://tailwindcss.com/docs/gradient-color-stops
